xml/System.Composition.Hosting/ContainerConfiguration.xml

```diff
@@ -27,8 +27,7 @@
   </Attributes>
   <Docs>
     <summary>Configures and constructs a lightweight container.</summary>
-    <remarks>
-    </remarks>
+    <remarks>To be added.</remarks>
```

Comment:
Do we have to put `To be added.` here? Is it not possible to leave it blank? Is that a special placeholder that isn't shown in the web?

I don't think anyone is actually going to add remarks here.

Reply:
It's special placeholder text that isn't shown in the web.

I think if you leave it blank, it will get re-added the next time we run the CI job to update APIs.